Calibration/PCCAlCaRecoProducers/interface/AlcaPCCProducer.h

```diff
@@ -0,0 +1,55 @@
+#ifndef Calibration_PCCAlCaRecoProducers_AlcaPCCProducer_h
+#define Calibration_PCCAlCaRecoProducers_AlcaPCCProducer_h
+
+/**_________________________________________________________________
+   class:   AlcaPCCProducer.h
+   package: Calibration/PCCAlCaRecoProducers
+   
+
+ authors:Sam Higginbotham (shigginb@cern.ch) and Chris Palmer (capalmer@cern.ch) 
+
+________________________________________________________________**/
+
+
+// C++ standard
+#include <string>
+// CMS
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDProducer.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "DataFormats/TrackerRecHit2D/interface/SiPixelRecHitCollection.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+#include "DataFormats/Luminosity/interface/PCC.h"
+
+class AlcaPCCProducer : public edm::one::EDProducer<edm::EndLuminosityBlockProducer,
```

Comment:
both twikis have an example.
I think that edm::stream will match the current implementation a bit more easily.
We should avoid edm::one if possible